homeassistant/components/arve/sensor.py

```diff
@@ -0,0 +1,110 @@
+"""Support for Arve devices."""
+
+from homeassistant.components.sensor import SensorDeviceClass, SensorEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+    CONCENTRATION_PARTS_PER_MILLION,
+    PERCENTAGE,
+    UnitOfTemperature,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import ArveCoordinator
+from .entity import ArveDeviceEntity, ArveDeviceEntityDescription
+
+SENSORS: tuple[ArveDeviceEntityDescription, ...] = (
+    ArveDeviceEntityDescription(
+        key="CO2",
+        translation_key="co2",
+        native_unit_of_measurement=CONCENTRATION_PARTS_PER_MILLION,
+        device_class=SensorDeviceClass.CO2,
+        value_fn=lambda arve_data: getattr(arve_data, "co2"),
```

Comment:
if you defined your value_fn with a default of `lambda arve_data, key: getattr(arve_data, lower(key))` and also pass the key, you can avoid defining this for every sensor